### PR TITLE
[MERGE] l10n_it_edi_sdicoop: remove module and merge its content into l10n_it_edi

### PR DESCRIPTION
**Pull request by dbkosky**
The email edi system defined in l10n_it_edi has been superceeded by the
edi system defined in l10n_it_edi_sdicoop. Therefore, the content of
l10n_it_edi_sdicoop can be merged into l10n_it_edi and the module can be
removed.

task-id: 2612853

### CODE REVIEW

File: addons/l10n_it_edi/models/account_invoice.py [hunk 1]
```diff
@@ -273,6 +233,13 @@ def _compose_multi_info_message(self, tree, element_tags):
                     output_str += "<li>%s: %s</li>" % (element.tag, text)
         return output_str + "</ul>"
 
+    def send_pec_mail(self):
+        self.ensure_one()
+        # OVERRIDE
+        # With SdiCoop web-service, no need to send PEC mail.
+        # Set the state to 'other' because the invoice should not be managed par l10n_it_edi.
+        self.l10n_it_send_state = 'other'
```
Comment (william-andre) [inline addons/l10n_it_edi/models/account_invoice.py:241]: Looks like it should be removed

**Review by william-andre — COMMENTED** [review]
Why are we removing a feature? There is no explanation on the PR nor in the internal task

File: addons/l10n_it_edi/models/account_edi_format.py [hunk 2]
```diff
@@ -615,3 +827,42 @@ def _import_fattura_pa(self, tree, invoice):
...
+        :returns:        A dictionary.
+        * message:       Message from fatturapa.
+        * transactionId: The fatturapa ID of this request.
+        * error:         An eventual error.
+        * error_level:   Info, warning, error.
+        '''
+        ERRORS = {
+            'EI01': {'error': _lt('Attached file is empty'), 'blocking_level': 'error'},
+            'EI02': {'error': _lt('Service momentarily unavailable'), 'blocking_level': 'warning'},
+            'EI03': {'error': _lt('Unauthorized user'), 'blocking_level': 'error'},
```
[the remark this answers is not in the document]
Comment (jco-odoo) [reply addons/l10n_it_edi/models/account_edi_format.py:1075]: Just lazy loading, should be fine

File: addons/l10n_it_edi/models/account_edi_format.py [hunk 3]
```diff
@@ -775,3 +774,315 @@ def _import_fattura_pa(self, tree, invoice):
...
+        if self.code != 'fattura_pa':
+            return super()._is_required_for_invoice(invoice)
+
+        is_self_invoice = self._l10n_it_edi_is_self_invoice(invoice)
+        return (
+            (invoice.is_sale_document() or (is_self_invoice and invoice.is_purchase_document()))
+            and invoice.country_code == 'IT'
+        )
+
+    def _export_as_xml(self, invoice):
```
Comment (jco-odoo) [inline addons/l10n_it_edi/models/account_edi_format.py:808]: We can keep it, but it is actually the same as _get_invoice_edi_content @dbkosky

**Review by jco-odoo — APPROVED** [review]

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by jco-odoo** [comment]
@robodoo delegate+

**Comment by jco-odoo** [comment]
@dbkosky

**Comment by dbkosky** [comment]
@robodoo r+

**Comment by dbkosky** [comment]
@robodoo r+

**Comment by dbkosky** [comment]
@robodoo retry

**Comment by dbkosky** [comment]
@robodoo r+

**Comment by dbkosky** [comment]
@robodoo retry

**Comment by dbkosky** [comment]
@robodoo r-

**Comment by dbkosky** [comment]
@robodoo r+

**Comment by dbkosky** [comment]
@robodoo r+

**Pull request closed without merging**